### ISSUE TITLE
Inconsistency in cover layouts

### ISSUE THREAD

**Issue by skleinfeldt** [jazkarta/jazkarta.com#107]
On some cover pages (agile, services), the text in the full width rich text tile lines up on the left with the edge of the basic tile below it, and on some (about, training) it doesn't. I couldn't find any difference in cover settings that I can control. It should always line up.

**Comment by davisagli** [comment]
Looking at the about page, the full-width tile that is too wide has a col-md-16 class...i.e. it's configured to span 16 columns (leftover from a different grid system?). Try adjusting the width to make sure it's set at 12 columns which is supposed to be the maximum.

**Comment by skleinfeldt** [comment]
Can I adjust the width in the cover layout tool? Or do you mean I should remove and re-add that row? 

I don't understand what I could have done to mess it up....

**Comment by davisagli** [comment]
I don't think you did anything to mess it up, the layout may have just been set up before we had the collective.cover grid settings right.

I was going to show you how to fix it, but when I try to edit a column's width the modal does not work correctly:

![screen shot 2015-10-19 at 4 35 18 pm](https://cloud.githubusercontent.com/assets/49014/10594212/893eb0c8-767f-11e5-9d6f-5eac6124d8bc.png)

Any ideas @cewing or @alecpm?

**Comment by skleinfeldt** [comment]
I noticed that modal weirdness too but I don't see it anymore on the About layout page. 

The modal doesn't show me any information about "Actual column size" - is that a permission thing? 

[There is another minor display problem on the Rich Text modal when Vertically Centered Text is selected (the gear is off the edge), which you can see on the About layout page, don't know if that's related.]

**Comment by davisagli** [comment]
Still doesn't work for me. I'm talking about the modal that you get to via the left/right arrow icon on the Layout page, not any of the gear icons.

**Comment by skleinfeldt** [comment]
Oh that modal - I don't know that I've ever used that one before... I did see a similar funky display issue on one of the gear modals earlier (I think) but can't find it now...

**Comment by skleinfeldt** [comment]
I have fixed (I think) all the covers where alignment was an issue, but deleting the offending row and replacing with another row. So this is no longer a priority. But the business with the modals not behaving right is still out there...
